CHANGELOG.md

```diff
@@ -2,11 +2,11 @@
 
 All notable changes to this project will be documented in this file, in reverse chronological order by release.
 
-## 1.0.2 - TBD
+## 1.1.0 - TBD
```

Comment:
as it is a feature PR should be to develop, and there we have placeholder for 1.1.0 release